doc/tutorial/04_search/index.md

```diff
@@ -0,0 +1,339 @@
+# Search with Raptor {#tutorial_search}
+
+You will learn how to searche a Raptor index using one or more sequences queries.
+
+\tutorial_head{Easy, 30 min, \ref tutorial_index, }
+
+[TOC]
+
+# Search an index using one or more sequences queries
+
+## A first search
+
+Now we want to have a look at Raptors second method: `raptor search`.
+
+We have already learned how to build an index in the previous tutorial on the `raptor index` and now we want to learn
+how to search in it.
+But before we delve deeper into the various possibilities and parameters. We can start a first search with the indexes
+we have already created.
+
+For this we just need the index and a query to run
+`raptor search --index raptor.index --query query.fasta --output search.output`.
+
+A result from such a call can look as follows:
+```text
+#0	example_data/64/bins/bin_00.fasta
+#1	example_data/64/bins/bin_01.fasta
+...
+#62	example_data/64/bins/bin_62.fasta
+#63	example_data/64/bins/bin_63.fasta
+#QUERY_NAME	USER_BINS
+0	0
+1	0
+2	0
+3	0
+4	0
+16384	1
+...
+1015812	62
+1032192	63
+1032193	63
+1032194	63
+1032195	63
+1032196	63
+```
+The output starts with a header section (lines starting with `#`). The header maps a number to each input file.
+After the header section, each line of the output consists of the read ID (in this example these are numbers) and the
+corresponding bins in which they were found.
+
+\assignment{Assignment 1: Create a first search on our created example files and their raptor index}
+From the first assignment in the last tutorial your directory should look like this:
+```bash
+tmp$ ls
+all_paths.txt   mini_1.fasta    mini_2.fasta    mini_3.fasta    raptor.index    raptor2.index    ...
+```
+Lets search for the queries `CGCGTTCATT` amd `CGCGTCATT` with the first two indexes, with creating a `search.output` and
```

Comment:
The query is shorter than k (19 for the first index, 4 for the second).
I don't check this (you usually search long sequences), but it needs to be >=k.